### PR TITLE
Add kir4h repository

### PR DESCRIPTION
**Pull request by kir4h**
Signed-off-by: Andres Alvarez <1671935+kir4h@users.noreply.github.com>

The motivation for this PR is to make a chart available for registry-creds. 

Originally I submitted a [PR](https://github.com/helm/charts/pull/13660) to helm/charts/incubator, but on that thread @maorfr  suggested to host the chart instead.

Any feedback is welcome as this is my first chart and also my first approach to hosting a helm repository.

### CODE REVIEW

File: repos.yaml [hunk 1]
```diff
@@ -266,6 +266,11 @@ repositories:
     maintainers:
       - email: kerkhove.tom@gmail.com
         name: Tom Kerkhove
+  - name: kir4h
+    url: https://kir4h.github.io/charts
+    maintainers:
+      - name: Andres Alvarez 
+        email: 1671935+kir4h@users.noreply.github.com
```
Comment (mattfarina) [inline repos.yaml]: We need a real routable to contact you in case we need to contact you about something related to the repository.

**Review by mattfarina — CHANGES_REQUESTED** [review]

**Comment by mattfarina** [comment]
@kir4h Thank you for the PR. Unfortunately, we need a real method to contact you in case there is  a problem with the repository.

File: repos.yaml (hunk 1 above)
Comment (kir4h) [reply repos.yaml]: Added routable e-mail

**Comment by maorfr** [comment]
nicely done @kir4h ! :)

**Review by mattfarina — APPROVED** [review]

**Pull request merged**